packages/facade/src/apis/facade.ts

```diff
@@ -325,6 +326,13 @@ export class FUniver {
         return this._injector.createInstance(FSheetHooks);
     }
 
+    /**
+     * Get hooks
+     */
+    getHooks() {
```

Comment:
The name should be more specific, like `getLifecycleHooks`. Unless you're going to bind other types of hooks o `f-hook`. (And what are other possible books?)

Reply:
I have considered this, and it is indeed worth discussing.

The current design is divided into 4 types of hooks, FHooks general hooks, FSheetHooks/FDocHooks/FSlideHooks hooks for specific document types

Although the role of hooks can be replaced by listening to Command in most scenarios, hooks are more semantic and more popular. In the future, we may refer to hansontable to add some hooks https://handsontable.com/docs/javascript-data-grid/api/hooks/
Add afterDispose, afterCopy, afterPaste... to FHooks

This brings up a management issue. Should the internal distinction of the 4 types of hooks be refined?
If it is necessary to refine, it can be divided into
FLifecycleHooks: including life cycle, dispose event
FClipboardHooks: clipboard hook
FRowHooks
FColumnHooks
FCellHooks
etc.

Which solution is better?

Reply:
OK. Let's merge all hooks in this class. Maybe it's better for fresh users to get what they want.